crates/python/qcs_sdk/_execution_data.pyi

```diff
@@ -0,0 +1,50 @@
+"""
+Do not import this file, it has no exports.
```

Comment:
I think the properties in this file also have setters

Reply:
I figured I'd omit them as these should generally be treated as immutable, but I can add them.